app/assets/javascripts/app/views/help_view.js

```diff
@@ -187,8 +197,23 @@ app.views.Help = app.views.StaticContentView.extend({
     e.preventDefault();
   },
 
+  chat: function(e){
+    this.renderStaticSection("chat", "faq_chat", this.CHAT_SUBS);
+    this.menuClicked(e);
+
+    e.preventDefault();
+  },
+
   linkHtml: function(url, text) {
     return "<a href=\"" + url + "\" target=\"_blank\">" + text + "</a>";
+  },
+
+  linkImage: function(src, alt){
+    return '<div class="help-image"><img src="images/' + src + '" alt="' + alt + '"/></div>';
```

Comment:
In most cases (when there are less than 20 contacts in the aspect) there will be also the mail icon in the control bar. The really important thing is the chat icon. I am still not convinced that we should add an image here. Why don't we describe where you will find the button (Go to the contacts page, select an aspect and click on the chat icon in the icon bar under the header to enable chat for the aspect) and add the original entypo icon?